x-pack/plugins/synthetics/common/rules/synthetics/translations.ts

```diff
@@ -11,48 +11,51 @@ export const SyntheticsMonitorStatusTranslations = {
   defaultActionMessage: i18n.translate(
     'xpack.synthetics.alerts.syntheticsMonitorStatus.defaultActionMessage',
     {
-      defaultMessage:
-        'The monitor {monitorName} checking {monitorUrl} from {locationName} last ran at {checkedAt} and is {status}. The last error received is: {lastErrorMessage}.',
+      defaultMessage: `"{monitorName}", {locationName} - {monitorType} monitor, is {status} at {checkedAt}.\nThe error received is: "{lastErrorMessage}".\n{linkMessage}`,
```

Comment:
i think message should be a sentence `{name}, {locatin} - {type}` format seems weird to me.

Reply:
cc: @florent-leborgne

Reply:
There are so many parameters that it's hard to articulate all of them into 1 readable sentence, especially because the location names can be quite lengthy and the monitor name can be anything.
Here are a couple of suggestions:

`{Name} is {status} from {Location}.\n Here are your monitor details:\n
Monitor name: {name}\n
URL: {url}\n
Monitor type: {type}\n
Checked at: {checked at}\n
From: {location}
Error received: {errorMessage}\n`


`Your {Name} {type} monitor is {status} from {location}. It failed at {checked at}.\n The last error received is...`